sdk/search/search-documents/test/internal/synonymMap.txt

```diff
@@ -0,0 +1,2 @@
+United States, United States of America => USA
+Washington, Wash. => WA
```

Comment:
Can we add a third line to this test to catch the "only replace a single \r" bug?

Reply:
@xirzec Added the empty case also now

Reply:
I don't see it in the latest revision?

Reply:
I added an empty line at the end of the test file. It is not showing up in the diff.